app/assets/stylesheets/_card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  border-radius: 0;
+  box-shadow: 0 1px 1px rgba(0,0,0,0.1);
```

Comment:
This color should be on a variable, SASS allows you to do `rgba($some-color, 0.1)` 😄